tests/MongoDB.Driver.Tests/IndexKeysDefinitionBuilderTests.cs

```diff
@@ -186,9 +235,9 @@ public void Text_Typed()
 
         private void Assert<TDocument>(IndexKeysDefinition<TDocument> keys, string expectedJson)
         {
-            var renderedSort = Render<TDocument>(keys);
+            var rendered = Render<TDocument>(keys);
```

Comment:
Fwiw, the type argument is redundant. I admit that changing it might add some unwanted noise to this PR, but if we're changing this line anyways, perhaps it doesn't hurt to remove the type argument?

Reply:
Done